client/multi-currency/blocks/currency-switcher.js

```diff
@@ -0,0 +1,318 @@
+				symbol,
+				flag,
+				fontSize,
+				fontLineHeight,
+				fontColor,
+				border,
+				borderRadius,
+				borderColor,
+				backgroundColor,
+			},
+			setAttributes,
+		} = props;
+
+		// eslint-disable-next-line react-hooks/rules-of-hooks
+		const { enabledCurrencies } = useEnabledCurrencies();
+		// eslint-disable-next-line react-hooks/rules-of-hooks
+		const { isLoading } = useCurrencies();
+		const enabledKeys = enabledCurrencies
+			? Object.keys( enabledCurrencies )
+			: [];
+
+		// In case there is a problem retrieving enabled currencies.
+		const placeholders = [
+			{ id: 'usd', code: 'USD', flag: '🇺🇸', symbol: '$' },
+			{ id: 'eur', code: 'EUR', flag: '🇪🇺', symbol: '€' },
+			{ id: 'jpy', code: 'JPY', flag: '🇯🇵', symbol: '¥' },
+		];
+
+		const styles = {
+			div: {
+				lineHeight: fontLineHeight,
+			},
+			select: {
+				fontSize: fontSize,
+				color: fontColor,
+				backgroundColor: backgroundColor,
+				borderWidth: border ? '1px' : '0px',
+				borderColor: borderColor,
+				borderRadius: borderRadius,
+			},
+		};
+
+		// eslint-disable-next-line react-hooks/rules-of-hooks
+		const blockProps = useBlockProps();
+
+		const onChangeFlag = ( newFlag ) => {
+			setAttributes( { flag: newFlag } );
+		};
+
+		const onChangeSymbol = ( newSymbol ) => {
+			setAttributes( { symbol: newSymbol } );
+		};
+
+		const onChangeFontSize = ( newFontSize ) => {
+			setAttributes( {
+				fontSize: newFontSize === undefined ? 11 : newFontSize,
+			} );
+		};
+
+		const onChangeFontLineHeight = ( newFontLineHeight ) => {
+			setAttributes( {
+				fontLineHeight:
+					newFontLineHeight === undefined
+						? 'normal'
+						: newFontLineHeight,
+			} );
+		};
+
+		const onChangeFontColor = ( newFontColor ) => {
+			setAttributes( {
+				fontColor:
+					newFontColor === undefined ? '#000000' : newFontColor,
+			} );
+		};
+
+		const onChangeBorder = ( newBorder ) => {
+			setAttributes( {
+				border: newBorder === undefined ? false : newBorder,
+			} );
+		};
+
+		const onChangeBorderRadius = ( newBorderRadius ) => {
+			setAttributes( {
+				borderRadius:
+					newBorderRadius === undefined ? 3 : newBorderRadius,
+			} );
+		};
+
+		const onChangeBorderColor = ( newBorderColor ) => {
+			setAttributes( {
+				borderColor:
+					newBorderColor === undefined ? '#000000' : newBorderColor,
+			} );
+		};
+
+		const onChangeBackgroundColor = ( newBackgroundColor ) => {
+			setAttributes( {
+				backgroundColor:
+					newBackgroundColor === undefined
+						? 'transparent'
+						: newBackgroundColor,
+			} );
+		};
+
+		return (
+			<div { ...blockProps }>
+				<InspectorControls key="setting">
+					<PanelBody
+						title={ __(
+							'Multi-Currency Settings',
+							'woocommerce-payments'
+						) }
+					>
+						<a
+							href={
+								'/wp-admin/admin.php?page=wc-settings&tab=wcpay_multi_currency'
+							}
+							target="_blank"
+							rel="noreferrer"
+						>
+							Adjust and edit your Multi-Currency settings
```

Comment:
The translation wrapper is missing here.

Reply:
Added, good catch! https://github.com/Automattic/woocommerce-payments/pull/2883/commits/17dc474dceb59d54cc63835d499c936d3bcab5e4